src/views/ideas/l10n.json

```diff
@@ -0,0 +1,69 @@
+    "ideas.starterProjectsButton": "Explore Starter Projects",
+    "ideas.tryTheTutorial": "Try the tutorial",
+    "ideas.codingCards": "Coding Cards",
+    "ideas.educatorGuide": "Educator Guide",
+
+    "ideas.tttHeader": "Things to Try",
+    "ideas.tttBody": "What do you want to make with Scratch? For each activity, you can try the <strong>Tutorial</strong>, download a set of <strong>Activity Cards</strong>, or view the <strong>Educator Guide</strong>.",
+    "ideas.cardsHeader": "Get the Entire Collection of Activity Cards",
+    "ideas.cardsBody": "With the Scratch Activity Cards, you can learn to create interactive games, stories, music, animations, and more!",
+    "ideas.cardsPurchase": "Purchase Printed Set",
+    "ideas.starterProjectsBody": "You can play with Starter Projects to get ideas for making your own projects.",
+    "ideas.starterProjectsPlay": "Play with Starter Projects",
+    "ideas.desktopEditorHeader": "Scratch Desktop Download",
+    "ideas.desktopEditorBody": "To create projects without an Internet connection, you can <a href=\"/download\">download Scratch Desktop</a>.",
+    "ideas.questionsHeader": "Questions",
+    "ideas.questionsBody": "Have more questions? See the <a href=\"/info/faq\">Frequently Asked Questions</a> or visit the <a href=\"/discuss/7/\">Help with Scripts Forum</a>.",
+    "ttt.tutorial": "Tutorial",
+    "tile.guides": "See Cards and Guides",
+    "tile.tryIt": "Try It",
+    "ttt.placeholder": "Placeholder text",
+    "ttt.tutorialSubtitle": "Find out how to make this project using a step-by-step tutorial in Scratch.",
+    "ttt.activityTitle": "Activity Cards",
+    "ttt.activitySubtitle": "Explore new coding ideas using this set of illustrated cards you can print out.",
+    "ttt.educatorSubtitle": "Use this educator guide to plan and lead a one-hour Scratch workshop.",
+    "ttt.open": "Open",
+    "ttt.MakeItFlyTitle": "Make It Fly",
+    "ttt.MakeItFlyDescription": "Animate the Scratch Cat, The Powerpuff Girls, or even a taco!",
+    "ttt.AnimateYourNameDescription": "Animate the letters of your username, initials, or favorite word.",
+    "ttt.RaceTitle": "Race to the Finish",
+    "ttt.RaceDescription": "Make a game where two characters race each other.",
+    "ttt.HideAndSeekTitle": "Hide and Seek",
+    "ttt.HideAndSeekDescription": "Make a hide-and-seek game with characters that appear and disappear.",
+    "ttt.FashionTitle": "Fashion Game",
+    "ttt.FashionDescription": "Make a game where you dress a character with different clothes and styles.",
+    "ttt.PongTitle": "Pong Game",
+    "ttt.PongDescription": "Make a bouncing ball game with sounds, points, and other effects.",
+    "ttt.DanceTitle": "Let's Dance",
+    "ttt.DanceDescription": "Design an animated dance scene with music and dance moves.",
+    "ttt.CatchTitle": "Catch Game",
+    "ttt.CatchDescription": "Make a game where you catch things falling from the sky.",
+    "ttt.VirtualPetTitle": "Virtual Pet",
```

Comment:
These all stick around as things translators have to translate. It's not a big deal for the folks that have already done the tips page, but for new languages (e.g., igbo), it will be extra work they don't really need to do.  We should remove any that we know we're not going to use. It would be ok to keep the ones that will have a new tutorial eventually.

Reply:
good point, removed most of the unused ones